facility-all-programs.ipynb

```diff
@@ -226,7 +226,10 @@
    "metadata": {},
    "outputs": [],
    "source": [
-    "#### THIS WILL BE EXPANDED AND REWORKED TO ACTUALLY DRAW FROM THE PROGRAM-SPECIFIC SPREADSHEETS WE LOAD IN A CELL BELOW\n",
+    "#### DELETE IN MAIN BRANCH\n",
```

Comment:
Yea, this is a product of my limited understanding of GH. I wanted this in master so that @danpascucci could work from it. I thought that if I removed it from this branch right now that would lead to merge conflicts for any work he's doing right now...Plan was to move it out of master after we pull his branch in.

Reply:
Oh I see. Your approach seems reasonable/maybe we should also make an issue to delete pending merge of @danpascucci's branch so that we have it documented somewhere other than code comments